runner/utils/driver/docker/docker.go

```diff
@@ -32,11 +32,12 @@ type Docker struct {
 	artifactAddr string
 	secretPath   string
 	hostFilePath string
+	clusterName  string
```

Comment:
In some places you call this `clusterID` and in some you call it `clusterName`. Lets be consistent. Include the parameter names

Reply:
so cluster name is a small name that the user provides, while cluster-id is a longer name that we use to uniquely identify each cluster. Thought we pass cluster-id to the runner through env variable we use cluster-name to set up different services within the runner, that is why we split cluster-id

Reply:
Have a look at your generator functions, you are passing `clusterName` and referring to it as `id` or `clusterID` in the function